pkgs/development/python-modules/mawk/default.nix

```
@@ -0,0 +1,24 @@
+{ python3Packages, lib }:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```
Do not use python3Packages when defining Python modules. Add `buildPythonPackage` etc. to the argument list above